abcpy/modelselections.py

```diff
@@ -0,0 +1,259 @@
+from abc import ABCMeta, abstractmethod
+import numpy as np
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+
+
+
+class ModelSelections(metaclass = ABCMeta):
+    """This abstract base class defines a model selection rulr of how to choose model from a set of models
```

Comment:
Fixed also abcpy.rst. 

But notice there is some warning regarding uniform _init_ in master. I don't understand that, please check that after merging.